### ISSUE TITLE
Confirm dialog with appendTo="body" throws error

### ISSUE THREAD

**Issue by anaq1** [primefaces/primeng#6384]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
https://stackblitz.com/edit/github-qqwfee?file=src/app/app.component.html

**Current behavior**

Adding appendTo="body" to p-confirmdialog leads to this error:
```
ConfirmDialog.html:2 ERROR TypeError: Cannot read property 'style' of undefined
    at ConfirmDialog.push../node_modules/primeng/components/confirmdialog/confirmdialog.js.ConfirmDialog.moveOnTop (confirmdialog.js:136)
    at ConfirmDialog.push../node_modules/primeng/components/confirmdialog/confirmdialog.js.ConfirmDialog.onAnimationStart (confirmdialog.js:67)
    at Object.eval [as handleEvent] (ConfirmDialog.html:4)
    at handleEvent (core.js:10258)
    at callWithDebugContext (core.js:11351)
    at Object.debugHandleEvent [as handleEvent] (core.js:11054)
    at dispatchEvent (core.js:7717)
    at core.js:8161
    at animations.js:169
    at Array.forEach (<anonymous>)
```
Without appendTo="body" the error doesn't occur.

In stackblitz example, the confirmation dialog is at least shown, but you can see the error in console. In my app, the dialog is not shown, only removing the appendTo="body" helps.

**Expected behavior**
No error.

**Minimal reproduction of the problem with instructions**
Create a p-confirmation with appendTo="body".

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:** Windows 10 64-bit

* **Angular version:** 6.1.3

* **PrimeNG version:** 6.1.2

* **Browser:** [Chrome 68.0.3440.106]
 
* **Language:** [TypeScript 2.9.2]

* **Node (for AoT issues):** `node --version` =  9.1.0

**Comment by andyg3** [comment]
Duplicate #6362

**Comment by cagataycivici** [comment]
Fixed in 6.1.3
